config/application.rb

```diff
@@ -17,6 +17,8 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+APP_NAME = 'login.gov'.freeze
```

Comment:
Is this intended to be plain login.gov or the partner dashboard?

Reply:
It's intended to be 'login.gov'.  I created the constant while converting `app/views/user_mailer/welcome_new_user.html.slim` to `app/views/user_mailer/welcome_new_user.html.erb`.  I could change the constant name, but as it doesn't have much value beyond that template right now, maybe I should remove the constant and use a presenter?  It might be overkill, but it should avoid this confusion.  Thanks for pointing this out.

Reply:
I think that's misleading? This is the Dashboard app, why does the email say "welcome to login.gov?" Shouldn't it be "welcome to the login.gov integration dashboard" or something? login.gov sends its own emails, we need to make sure they're easy to differentiate, right? These are separate apps

Reply:
Where was my head?  The slim template was:
```
html
  head
    meta content='text/html; charset=UTF-8' http-equiv='Content-Type'
  body

  p Welcome to #{root_url}!

  p
    =link_to root_url, root_url

  p
    strong PLEASE DO NOT REPLY TO THIS MESSAGE
```
The link_to was what led me to talk to you about possibly doing something like `URI.parse(root_url).domain`. Now, I'm not sure about it.  I think I should just get rid of `APP_NAME` and keep `root_url` or change it to `root_path` (isn't that more appropriate for views), no?

Reply:
The other templates use `Rails.application.config.app_name` which is the Dashboard URL, so let's stick with that?